data.json

```diff
@@ -23914,7 +23965,10 @@
     },
     "yandex-browser-beta": {
         "linux": {
-            "root": "yandex-browser-beta"
+            "root": "yandex-browser-beta",
+            "symlinks": [
+                "yandex-browser"
+            ]
```

Comment:
To make our future lives easier, lets switch this round now so the icon and root are both `yandex-browser` and the symlink is `*-beta`